cmd/agent/main.go

```diff
@@ -70,7 +70,6 @@ func main() {
 			err = errors.Join(
 				clean.DuplicateBindings(nil),
 				clean.OrphanBindings(nil),
-				clean.OrphanCatalogBindings(nil),
```

Comment:
Related code from `pkg/data/management/bindings_cleanup.go` seem to no longer be needed:

```golang
	orphanCatalogBindingsCleanupKey = "CleanupOrphanCatalogBindingsDone"

...

	err = cleanupSpecificOrphanedBindings(scaledContext, wContext, orphanCatalogBindingsCleanupKey)
	if err != nil {
		logrus.Errorf("failed to cleanup orphan catalog bindings")
	}
```